airflow/providers/amazon/aws/transfers/gcs_to_s3.py

```diff
@@ -31,6 +34,9 @@
     from airflow.utils.context import Context
 
 
+google_provider_version = version("apache-airflow-providers-google")
```

Comment:
We have to also handle the case where google provider is installed from sources (editable install). Better check is where we do `from apache.airflow.providers.google import __version__`.

It's a relatively new addition - and we have to `try/exceptImportError` for it and handle "unknown" version - but it's been addedd in 10.1.0 version of Google, so checking for '< 10.3.0 or unknown` should be OK.

Reply:
It's a good idea, I just implemented it, could you take a second look?